config/initializers/rack_timeout.rb

```diff
@@ -1,11 +1,15 @@
-Rack::Timeout.service_timeout = 30
+# Rack::Timeout.service_timeout = 30
+
+# insert middleware wherever you want in the stack, optionally pass
+# initialization arguments, or use environment variables
+Rails.application.config.middleware.insert_before Rack::Runtime, Rack::Timeout, service_timeout: 30
```

Comment:
I don't think this is something we've had to do elsewhere, so I'd like to get to the bottom of why we had to do it here -- and maybe then we can also undo [this change](https://github.com/Betterment/test_track/pull/182/files#r1113689994) as well.